src/client/styles/_editor.scss

```diff
@@ -8,6 +8,11 @@
         background: $background-color;
         color: $editor-gutter-color;
     }
+    .ace_gutter-cell.ace_breakpoint{ 
+        border-radius: 20px 0px 0px 20px;
+        /* Change the color of the breakpoint if you want */
+        box-shadow: 0px 0px 1px 1px #248c46 inset; 
+    }
```

Comment:
Can you include a screenshot of the visual changes in the PR description? (I think you can just drag/drop an image when you're editing the description)